examples/x86_64.swiftinterface

```diff
@@ -0,0 +1,511 @@
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct AddChatParticipantsRequest : Swift.Codable {
+  public let participants: [AzureCommunicationChat.ChatParticipant]
+  public init(participants: [AzureCommunicationChat.ChatParticipant])
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct DeleteChatMessageOptions : AzureCore.RequestOptions {
+  public let clientRequestId: Swift.String?
+  public let cancellationToken: AzureCore.CancellationToken?
+  public var dispatchQueue: Dispatch.DispatchQueue?
+  public var context: AzureCore.PipelineContext?
+  public init(clientRequestId: Swift.String? = nil, cancellationToken: AzureCore.CancellationToken? = nil, dispatchQueue: Dispatch.DispatchQueue? = nil, context: AzureCore.PipelineContext? = nil)
+}
+public struct ChatThreadsItemCollection : Swift.Codable {
+  public let value: [AzureCommunicationChat.ChatThreadItem]
+  public let nextLink: Swift.String?
+  public init(value: [AzureCommunicationChat.ChatThreadItem], nextLink: Swift.String? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct ListChatReadReceiptsOptions : AzureCore.RequestOptions {
+  public let maxPageSize: Swift.Int32?
+  public let skip: Swift.Int32?
+  public let clientRequestId: Swift.String?
+  public let cancellationToken: AzureCore.CancellationToken?
+  public var dispatchQueue: Dispatch.DispatchQueue?
+  public var context: AzureCore.PipelineContext?
+  public init(maxPageSize: Swift.Int32? = nil, skip: Swift.Int32? = nil, clientRequestId: Swift.String? = nil, cancellationToken: AzureCore.CancellationToken? = nil, dispatchQueue: Dispatch.DispatchQueue? = nil, context: AzureCore.PipelineContext? = nil)
+}
+public struct GetChatThreadPropertiesOptions : AzureCore.RequestOptions {
+  public let clientRequestId: Swift.String?
+  public let cancellationToken: AzureCore.CancellationToken?
+  public var dispatchQueue: Dispatch.DispatchQueue?
+  public var context: AzureCore.PipelineContext?
+  public init(clientRequestId: Swift.String? = nil, cancellationToken: AzureCore.CancellationToken? = nil, dispatchQueue: Dispatch.DispatchQueue? = nil, context: AzureCore.PipelineContext? = nil)
+}
+public struct CommunicationErrorResponse : Swift.Codable, Swift.Error {
+  public let error: AzureCommunicationChat.ChatError
+  public init(error: AzureCommunicationChat.ChatError)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct UpdateChatThreadRequest : Swift.Codable {
+  public let topic: Swift.String?
+  public init(topic: Swift.String? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct AzureCommunicationChatClientOptions : AzureCore.ClientOptions {
```

Comment:
Would you prefer to alias this to something shorter?  ChatClientOptions?

Reply:
ChatClientOptions makes sense, is this a small change on autorest side?